hax/hax/queue/__init__.py

```diff
@@ -84,16 +84,23 @@ def handle_ioq_stob_error(self, payload: Dict[str, Any]) -> None:
         self.planner.add_command(
             BroadcastHAStates(
                 states=[HAState(fid,
-                                status=ServiceHealth.FAILED)], reply_to=q))
+                                status=ObjHealth.FAILED)], reply_to=q))
         ids: List[MessageId] = q.get()
         self.herald.wait_for_any(HaLinkMessagePromise(ids))
 
     def to_ha_state(self, objinfo: Dict[str, str]) -> Optional[HAState]:
+        hastate_to_objstate = {
```

Comment:
@mssawant, can we update "drive-state" usage in utils/hare-drive-state

Reply:
This is for CORTX-22105 JIRA.

Reply:
Updated.